docs/tutorials/develop-with-ubuntu-wsl.md

```diff
@@ -53,95 +79,94 @@ PS C:\Users\username>
 username@pc:~$
 ```
 
-To exit the Ubuntu terminal, type the `exit` command and execute it by pressing
-<kbd>enter</kbd>, which will return you to the PowerShell prompt.
-
-```{tip}
-It is recommended to reboot your machine after this initial installation to
-complete the setup.
-```
+To exit the Ubuntu terminal at any time, type the `exit` command and execute it
+by pressing <kbd>enter</kbd>, which will return you to the PowerShell prompt.
 
-### Install a specific version of Ubuntu on WSL
 
+```{admonition} Different methods to install Ubuntu on WSL
+:class: tip
 There are multiple ways of installing Ubuntu on WSL, here we focus on using the
 terminal. For more detail on installation methods for Ubuntu on WSL, refer to
 our [dedicated installation guide](../howto/install-ubuntu-wsl2.md).
-
-To install Ubuntu 24.04 LTS, run the following command in a PowerShell terminal:
-
-```{code-block} text
-> wsl --install Ubuntu-24.04
 ```
 
-You'll see an indicator of the installation progress in the terminal:
+### Running multiple versions of Ubuntu
 
-```{code-block} text
-:class: no-copy
-Installing: Ubuntu 24.04 LTS
-[==========================72,0%==========                 ]
-```
+You can install multiple versions of Ubuntu on WSL. Each Ubuntu instance can
+then be used as a separate, self-contained development environment.
 
-```{note}
-WSL supports a variety of Ubuntu releases. Read our [reference on distributions
-of Ubuntu on WSL](../reference/distributions.md) for more information.
+```{code-block} text
+> wsl --install Ubuntu-22.04
 ```
 
-### Run a specific Ubuntu version
-
-Use `wsl -l -v` to list all your installed distros and the version of WSL that they are using:
+Use `wsl -l -v` to list all of your installed distros.
 
 ```{code-block} text
 :class: no-copy
   NAME            STATE           VERSION
-  Ubuntu          Stopped         2
+  Ubuntu-22.04    Stopped         2
 * Ubuntu-24.04    Stopped         2
 ```
 
-Two instances of Ubuntu are installed:
-
-1. The default Ubuntu version that was installed automatically when you installed WSL
-2. The numbered Ubuntu version that you installed manually
+```{admonition} What is version 2?
+:class: note
+This means that you are using WSL2, rather than WSL1.
+WSL2 is the default WSL on recent versions of Windows.
```

Comment:
```suggestion
WSL implements two different architectures for running 
Linux distributions: WSL1 and WSL2.
This means that you are using WSL2, rather than WSL1.
WSL2 is the default WSL on recent versions of Windows.
```

Reply:
If the user is questioning what's the VERSION 2 value, they likely don't know about WSL1 x WSL2, so for those we are still not answering their question.